docs/operate/get-started/other-hardware/_index.md

```diff
@@ -136,6 +145,9 @@ Authenticate your CLI session with Viam using one of the following options:
 
 {{< /expand >}}
 
+{{% /tablestep %}}
+{{< /table >}}
+
 The generator will create a folder containing stub files for your modular sensor component.
 In the next section, you'll customize some of the generated files to support your sensor.
 
```

Comment:
The Python tab below still needs tablestep formatting so it doesn't do this
![image](https://github.com/user-attachments/assets/64b96597-d010-40b4-ade7-1d91c73ef047)